### DISCUSSION TITLE
Remote router children RSSI values by CLI

### DISCUSSION THREAD

**Discussion by kevin-fisk** [openthread/openthread#8385] [Q&A] [answered]
Is there a CLI method to gain information on a remote router's children's RSSI? 

For example, if I use the CLI on the router with attached children, I can issue 'child table' to see the attached children. Based on the returned info, issuing 'child x' where x is the child ID from the table, I can see both Link Quality and RSSI. If I use the CLI on another router in the network, I can use a 'networkdiagnostic get' to see an attached child. 

However I'm seeking a CLI method (if there is one) to see both or either "Link Quality" or "RSSI" of the attached children. Is this possible?

**Comment by abtink** [comment, marked as answer]
@tekmar-kevinf, as you said we can get full `child table` of this device itself (which contains link quality info of each child), but I don't think we have a way today to get such info about another router. We can get some info using `network diagnostics` about the child table of another router but it does not provide RSSI or link quality info (it contains child id and timeout).

**Reply by kevin-fisk** [reply]
Thank you @abtink . I was 'hoping' I missed a way of reading the strength of the connectivity, and the mesh, on end devices from a leader for example.
